docs/api.md

```diff
@@ -340,6 +340,7 @@ articles.loadDump([
 
 > #### Notes
 >
+> - The use of `loadDump()` is deprecated.  Use `importBulk()` instead.
```

Comment:
Similarly, I'm not sure we need to add a note about a method that isn't documented here.